### DISCUSSION TITLE
Where to find IMX283 datasheet and registers

### DISCUSSION THREAD

**Discussion by watchever** [will127534/OneInchEye#16] [Q&A]
Hello, Does someone have the IMX283 registers list and could share it ?
I would lile to know if IMX283 support cumulative pixel binning.

Thanks for your help

**Comment by yassinebkr** [comment]
It is a pretty sensitive file, usually you can have it when you buy some cmos.

**Comment by yassinebkr** [comment]
Features
◆CMOS active pixel type pixels
◆Input clock frequency 6 to 27 MHz
◆MIPI Specifications (CSI-2 high-speed serial interface)
◆All-pixel scan mode
Horizontal/vertical 2/2-line binning mode
Horizontal/vertical 3/3-line binning mode
Vertical 2/9 subsampling binning horizontal 3 binning mode
Vertical 2/19 subsampling binning horizontal 3 binning mode
Vertical 2 binning horizontal 2/4 subsampling mode
◆High-sensitivity, low dark current, no smear, excellent anti-blooming characteristics
◆Vertical and horizontal arbitrary cropping function
◆Variable-speed shutter function (minimum unit: 1 horizontal period)
◆Low power consumption
◆H driver, V driver and I2C communication circuit on chip
◆CDS/PGA on chip: Gain +27 dB (step pitch 0.1 dB)
◆9-bit/10-bit/12-bit A/D conversion on chip
◆R, G, B primary color mosaic filters on chip
◆118-pin high-precision ceramic package
 That's all I have for you
